README.md

```diff
@@ -76,7 +76,7 @@ end
   end
 ```
 
-- Use empty lines between `def`s and to break up a method into logical paragraphs.
```

Comment:
My reading of this is that it's encouraging an empty line _between method blocks_ as well as within methods, i.e.
```ruby
def foo
  # ...
end

def bar
  # ...
end
```

How about changing it to something like...
> Use empty lines between method (`def`) blocks, and within methods to break them up into logical paragraphs

Reply:
Yeah I see it now, I was reading it differently. Will make the change.